src/components/Country/Country.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import Card from 'react-bootstrap/Card';
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+import { FaRegArrowAltCircleRight } from 'react-icons/fa';
+import './country.css';
+
+const Country = ({ name, flags }) => (
+  <Row xs={1} lg={1} className="g-4">
+    <Col>
+      <Card className="card-container">
+        <Link to={`/name/${name.common}`}>
+          <FaRegArrowAltCircleRight
+            style={{ width: '10px', height: '10px' }}
+            className="arrow-right"
+          />
+          <Card.Body>
+            <Card.Img variant="top" src={flags.png} />
+            <Card.Title>{`Country: ${name.common}`}</Card.Title>
+          </Card.Body>
+        </Link>
+      </Card>
+    </Col>
+  </Row>
```

Comment:
As requested by the [previous reviewer](https://github.com/kelomo2502/europe-quick-check/pull/4#discussion_r943055394), kindly make the following changes for your home page section:

- [ ] Kindly align your country and country name to the right side. This will be possible by breaking the card to two and aligning them to the right side so as to match the provided Behance design.
- [ ] There are some sections where the country name is overlapped by the details of the country card tat is below it. This is not the desired outcome that can be fixed by providing each card with a margin of top and bottom.
- [ ] Alo, kindly give your alternating country cards a different color. eg the first one gives it a different shade of pink and the other one leave it as it is. This way you will have met the requested change that was also mentioned by another reviewer.

**This is how your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/184100518-25e0a0f1-4531-4d38-9c7f-13456e2c1961.png)

**Note** the country name of Switzerland is overlapped by the details of the country below it and cannot be seen. I am on 100% zoom level.

**This is the desired Behance design as mentioned by the previous reviewer:**

![image](https://user-images.githubusercontent.com/86318284/184100846-3b74701b-264e-48eb-a881-d7f55ede37c5.png)